newswires/app/conf/SearchPresets.scala

```diff
@@ -1,7 +1,31 @@
 package conf
 
+import conf.Suppliers._
 import db.SearchParams
 
+// Increase max line length to improve readability of search presets
+// scalafmt: { maxColumn = 120 }
+
+object SearchPreset {
+  def apply(
+      supplier: String,
+      categoryCodes: List[String] = Nil,
+      categoryCodesExcl: List[String] = Nil,
+      text: Option[String] = None,
+      searchTerm: Option[SearchTerm] = None,
+      keyword: Option[String] = None,
+      keywordExcl: List[String] = Nil
+  ): SearchParams =
+    SearchParams(
+      text = searchTerm orElse text.map(SearchTerm(_, SearchConfig.Simple)),
```

Comment:
if I'm understanding the logic correctly, if both `searchTerm` and `text` are defined on for an instance of `SearchPreset`, then the `text` value will be discarded. Given this, would it be better if we made it so that you can only provide one or the other value when constructing a `SearchPreset`? (either via a function overload or something else?)